apis/Google.Storage.V1/Google.Storage.V1.Snippets/UrlSignerSnippets.cs

```diff
@@ -28,13 +28,13 @@
 namespace Google.Storage.V1.Snippets
 {
     [Collection(nameof(StorageSnippetFixture))]
-    public class SignedUrlUtilsSnippets
+    public class UrlSignerSnippets
     {
         private readonly string credentialsFilePath = GetCredentialsFilePath();
```

Comment:
Done (I assume you meant local but outside of the snippet, but if this assumption is wrong, please let me know).